# goal
string command
---
# result
bool success
---
# feedback
string status_initialization
string status_match_command
bool error_detected
